storybook/preview.js

```diff
@@ -1,7 +1,46 @@
+import React from 'react'
 import Vue from 'vue'
 import '~storybook';
 
 <% if (options.nuxtOptions.components) { %>import * as components from '../components';
-Object.keys(components).forEach(name => Vue.component(name, components[name]))<% } %>
+Object.keys(components).forEach(name => Vue.component(name, components[name]))<% } %>;
 
-export const parameters = <%= devalue(options.parameters) %>
\ No newline at end of file
+
+// based on: https://github.com/storybookjs/storybook/blob/master/addons/docs/src/frameworks/vue/prepareForInline.ts
+const COMPONENT = 'STORYBOOK_COMPONENT';
+const VALUES = 'STORYBOOK_VALUES';
+const prepareForInline = (storyFn, { args }) => {
+  const component = storyFn()
+  const el = React.useRef(null)
+  // FIXME: This recreates the Vue instance every time, which should be optimized
+  React.useEffect(() => {
```

Comment:
Is this only way to fix and listen for ? :confused:

Reply:
This is how docs addon work.
I don't see any workaround to refactor the code.